src/app/containers/LinkedData/index.jsx

```diff
@@ -19,11 +19,15 @@ const LinkedData = ({
     publishingPrinciples,
     defaultImage,
     noBylinesPolicy,
+    trustProjectParticipant,
```

Comment:
this prop naming makes me think of a string value e.g. `trustProjectParticipant = 'The Economist'`

what about renaming to `isTrustProjectParticipant` so we know we're dealing with a boolean?